packages/relay/src/create.ts

```diff
@@ -14,7 +14,7 @@ import { Relay, RelayCreateOptions, wakuGossipSub } from "./relay.js";
  * or use this function with caution.
  */
 export async function createRelayNode(
-  options: CreateNodeOptions & Partial<RelayCreateOptions>
```

Comment:
I think `Partial` should stay otherwise it will make all of `GossipsubOpts` mandatory for `createRelayNode`